db/app.js

```diff
@@ -1,9 +1,13 @@
 const express = require("express");
-const { getTopics, getArticle } = require("./controllers/topics.controller.js");
+const {
+  getTopics,
+  getArticleById,
+  getArticles,
+} = require("./controllers/controller.js");
 const app = express();
 const endpoints = require("../endpoints.json");
 
-app.use(express.json());
+// app.use(express.json());
```

Comment:
This is not required in your code until there is a POST or PATCH, commented out code should not be included in "production ready" code